deploy/olm-catalog/ibm-block-csi-operator-community/1.6.0/bundle-1.6.0.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM scratch
```

Comment:
please convert the [test tool](https://github.com/IBM/ibm-block-csi-operator/blob/3c6d21d87f618231937ae63bbd243ae694e0dbbc/Makefile#L25) to support the new format (and remove the `ocp` from the script name, since it can now be used for all catalogs)

Reply:
Done